conda_smithy/tests/test_lint_recipe.py

```diff
@@ -107,7 +107,7 @@ def test_test_section_with_recipe(self):
             lints = linter.lintify({}, recipe_dir)
             self.assertIn(expected_message, lints)
 
-            with open(os.path.join(recipe_dir, 'run_test.py'), 'w') as fh:
+            with io.open(os.path.join(recipe_dir, 'run_test.py'), 'w') as fh:
```

Comment:
Normally I don't. Unfortunately unicode issues have cropped up periodically with the linter. So it seemed like this might be the best way to fully address them.